boards/BMS/Src/App/states/App_AllStates.c

```diff
@@ -3,12 +3,17 @@
 #include "App_SetPeriodicCanSignals.h"
 #include "App_Accumulator.h"
 #include "App_SharedMacros.h"
+#include "App_SharedProcessing.h"
 
 #define MIN_CELL_VOLTAGE (3.0f)
 #define MAX_CELL_VOLTAGE (4.2f)
 #define DEFAULT_MIN_CELL_TEMP_DEGC (0.0f)
 #define DEFAULT_MAX_CELL_TEMP_DEGC (60.0f)
 
+#define MAX_POWER_LIMIT_W (78e3f)
+#define CELL_ROLL_OFF_TEMP_DEGC (40.0f)
+#define CELL_FULLY_DERATED_TEMP (60.0f)
```

Comment:
Prefer to use different constant for MAX here. Can change max param while testing without touching #defines for temperature code

Reply:
nevermind, I agree with u now